app/main.py

```diff
@@ -2,0 +1,36 @@
+from os import path, remove, makedirs
+
+
+def move_file(command: str) -> None:
+    command_words = command.split(" ")
+
+    if len(command_words) != 3 or command_words[0] != "mv":
+        raise ValueError(
+            f"{command} is invalid. Pls, use 'mv source destination'"
+        )
+
+    if not path.exists(command_words[1]):
+        raise FileNotFoundError(f"File {command_words[1]} not found")
+
+    _, source_file, destination = command_words
+
+    destination = path.normpath(destination)
+
+    if path.isdir(destination):
+        path_to_create = destination
+        new_filename = path.basename(source_file)
+    else:
+        path_to_create, new_filename = path.split(destination)
+
+    if path_to_create:
+        makedirs(path_to_create, exist_ok=True)
```

Comment:
The use of `makedirs` with `exist_ok=True` is correct and prevents errors when the directory already exists, addressing the checklist point about directory creation errors.